ch.sbb.scion.rcp.microfrontend.e3.app.demo.test/src/ch/sbb/scion/rcp/microfrontend/e3/app/demo/test/PopupCapabilityTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Project: RCS - Rail Control System
+ *
+ * © Copyright by SBB AG, Alle Rechte vorbehalten
+ */
+package ch.sbb.scion.rcp.microfrontend.e3.app.demo.test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.apache.log4j.BasicConfigurator;
+import org.eclipse.swt.widgets.MenuItem;
+import org.eclipse.swtbot.swt.finder.SWTBot;
+import org.eclipse.swtbot.swt.finder.finders.UIThreadRunnable;
+import org.eclipse.swtbot.swt.finder.keyboard.Keystrokes;
+import org.eclipse.swtbot.swt.finder.results.VoidResult;
+import org.eclipse.swtbot.swt.finder.waits.Conditions;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotMenu;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTable;
+import org.eclipse.ui.PlatformUI;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PopupCapabilityTest {
+
+  private static final int TIMEOUT = 10000;
+  private SWTBot bot;
+
+  // TODO: When creating more tests, this beforeAll code might be better in some base/delegate test class
+  @BeforeAll
+  static void beforeAll() {
+    // configure logging
+    BasicConfigurator.configure();
+
+    // might be required when running in headless mode
```

Comment:
Is this the comment? (I would not understand from this comment, immediately, that force activation is (may be) required as in headless configuration the shell is not automatically activated on opening - is this even correct? 😄 )

Reply:
I added the link to the eclipse-wiki